src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -689,6 +843,36 @@ bool parameterHasBadConditionalState(ParameterSymbol parameter, bool sense, Loca
                 return false;
             }
 
+            // TODO: can this be folded into 'getSlotForFieldOrProperty'?
+            int getSlotForFieldOrPropertyOrEvent(Symbol member)
+            {
+                if (member.Kind != SymbolKind.Field &&
+                    member.Kind != SymbolKind.Property &&
+                    member.Kind != SymbolKind.Event)
+                {
+                    return -1;
+                }
+
+                int thisSlot = -1;
```

Comment:
>thisSlot [](start = 20, length = 8)

This has changed in `getSlotForFieldOrProperty`. Please consider merging the two local functions.

Reply:
Done